libs/output-mapping/src/Storage/AbstractTableStructureModifier.php

```diff
@@ -22,44 +22,6 @@ public function __construct(
         $this->client = $clientWrapper->getTableAndFileStorageClient();
     }
 
-    /**
-     * @param array $keys
-     * @return array
-     */
-    protected function normalizeKeyArray(array $keys)
-    {
-        $logger = $this->logger;
-        return array_map(
-            function ($key) {
-                return trim($key);
-            },
-            array_unique(
-                array_filter($keys, function ($col) use ($logger) {
-                    if ($col !== '') {
-                        return true;
-                    }
-                    $logger->warning('Found empty column name in key array.');
-                    return false;
-                }),
-            ),
-        );
-    }
-
-    protected function modifyPrimaryKeyDecider(
```

Comment:
Nakonec jsem to vrátil do `PrimaryKeyHelper`, protoze se to pouziva i jinde nez jen v `TableStructureModifier`